Example/PrimerSDK/CreateClientToken.swift

```diff
@@ -17,3 +17,10 @@ struct CreateClientTokenRequest: Codable {
     let customerCountryCode: String?
```

Comment:
The transaction response on the developer's side, so they can parse the `requiredAction` and call the `resumeHandler`